src/components/AppHeader/index.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+import logo from '../../logo.svg';
+import {PageHeader} from 'react-bootstrap';
+
+const AppHeader = () => {
```

Comment:
if a function just has a return block statement we can ditch the return instruction and just leave what we want to return. AppHeader will be:

```javascript
// @flow
import React from 'react';
import logo from '../../logo.svg';
import {PageHeader} from 'react-bootstrap';

const AppHeader = () => <PageHeader bsClass="App-header">
  <img src={logo} className="App-logo" alt="logo" />
  Welcome to React
</PageHeader>

export default AppHeader;
```